src/App.jsx

```diff
@@ -1,7 +1,21 @@
 import ReactDOM from "react-dom/client";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AddPage from "../pages/AddPage/AddPage";
+import AllPage from "../pages/AllPage/AllPage";
+import CompletedPage from "../pages/CompletedPage/CompletedPage";
+import UnCompletedPage from "../pages/UnCompletedPage/UnCompletedPage";
+import ConfigurationPage from "../pages/ConfigurationPage/ConfigurationPage";
 
 const App = () => {
-
+  <BrowserRouter>
+    <Routes>
+      <Route path="/Add" element={<AddPage />} />
+      <Route path="/All" element={<AllPage />} />
+      <Route path="/Completed" element={<CompletedPage />} />
+      <Route path="/UnCompleted" element={<UnCompletedPage />} />
+      <Route path="/ConfigurationPage" element={<ConfigurationPage />} />
```

Comment:
Use `hyphen case` (`kebab case`) for urls.

Reply:
Done.